.github/workflows/build-pr.yml

```diff
@@ -0,0 +1,28 @@
+on:
+  pull_request:
+    branches:
+      - develop
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          lfs: true
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 'lts/*'
+          cache: npm
+      - name: Install Dependencies
+        run: npm ci
+      - name: Build
+        run: npm run build
+      - name: Upload Artifact
```

Comment:
Do we need to upload the artifact since we are not doing anything with it?

Reply:
It could be useful in debugging if the build fails, but you're right that we don't need it when the build succeeds. I've updated the workflow to reflect that.